### ISSUE TITLE
some error while cracking

### ISSUE THREAD

**Issue by Mehran** [deathsec/instagram-py#20]
**Select your issue type:** (check at least one)
- [x] Bug
- [ ] Question
- [ ] Suggestion
- [ ] Other (please describe):
i load 20k passlist and left it , when i back after hours i see below error 

```
Traceback (most recent call last):
  File "./instagram-py", line 29, in <module>
    InstagramPy.ExecuteInstagramPy()
  File "/root/instagram-py/InstagramPy/__init__.py", line 107, in ExecuteInstagramPy
    instagrampy.TryPassword()
  File "/root/instagram-py/InstagramPy/InstagramPyInstance.py", line 105, in TryPassword
    response_data = (r.json())['message']
  File "/usr/local/lib/python3.5/dist-packages/requests/models.py", line 892, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by deathsec** [comment]
**Did you use Instagram-Py Scripts or Command Line ?**

**Comment by Mehran** [comment]
Instagram-Py ` ./instagram-py -u username -pl pass.txt`

**Comment by deathsec** [comment]
you are lucky if you did not run that twice! , is this the first time executing this attack on this user ?

**Comment by Mehran** [comment]
Yes , first time , it will be good if it have resume

**Comment by deathsec** [comment]
you can try ``` ./instagram-py -c -vvv -u username -pl pass.txt ``` to resume the attack and possibly find the error!

**Comment by deathsec** [comment]
**Instagram-Py** does have a save feature in it!!

**Comment by Mehran** [comment]
im try` ./instagram-py -c -vvv -u username -pl pass.txt` but it start first passlist .

**Comment by deathsec** [comment]
I guess brute forcing instagram so many times makes the api to run crazy , I think I should handle this error in a different way!

**Comment by Mehran** [comment]
i can help you in any way you want .

**Comment by deathsec** [comment]
Its just a nit-pick , no worries... just have to add ``` except ``` in json decode and if it fails then retry the same password , I will do it later... though..

you see your save file with this command ``` ls ~/.instagram-py ``` its a json file , it will show you everything about the attack!

**Comment by deathsec** [comment]
okay I will leave this issue open until the next release , and I recommend you to try instagram-py scripts its better than modifying source files , you can use it to send message to telegram or your phone when the attack is successfully finished ( Checkout Twilio API if you want to send message or a call directly to your phone ) , Okay Hope the continue feature helps you until I fix this bug..

**Comment by deathsec** [comment]
This issue is fixed in Instagram-Py v2.0.1 , Upgrade to the latest version now!
```
$ sudo pip3 install instagram-py --upgrade
```
